css/style.css

```diff
@@ -0,0 +1,589 @@
+    background: url("https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 254px;
+  }
+
+  #sticky-footer {
+    margin: 0;
+    padding: 0;
+  }
+
+  .footer-copyright,
+  .social {
+    align-self: flex-end;
+    margin-right: auto;
+  }
+
+  #footer-content {
+    flex-direction: row;
+  }
+
+  .footer-copyright {
+    align-self: flex-end;
+  }
+}
+
+@media screen and (max-width: 991px) {
+  #middle-navbar,
+  nav.nav-navigation,
+  nav.navia {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  main {
+    margin-top: 20px;
+  }
+
+  nav.navia {
+    position: relative;
+    z-index: -1;
+  }
+
+  header.header-container {
+    position: -webkit-sticky; /* Safari */
+    position: sticky;
+    top: 0;
+  }
+
+  .top-header {
+    padding: 0;
+  }
+
+  #middle-navbar {
+    display: block;
+  }
+
+  .weather-icon {
+    width: 40px;
+  }
+
+  picture {
+    display: inherit;
+  }
+
+  /* This is for the middle navbar */
+
+  .nav-navigation {
+    display: flex;
+    justify-content: space-between;
+    height: 55px;
+  }
+
+  .navbar-nav1 {
+    display: flex;
+    flex-wrap: nowrap;
+    align-content: center;
+    align-items: stretch;
+    width: 90%;
+    height: 54px;
+  }
+
+  .navbar-nav1 li {
+    list-style-type: none;
+    padding: 3px;
+    flex-grow: 1;
+  }
+
+  ul.navbar-nav1 {
+    font-size: 1rem;
+    line-height: 24px;
+    font-weight: bold;
+    font-family: "Roboto Condensed", sans-serif;
+    color: black;
+  }
+
+  ul.navbar-nav1 li a {
+    color: black;
+    display: block;
+    padding: 15px 5px;
+    flex-grow: 1;
+  }
+
+  #nav-link-long {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .feature-section,
+  .story-section,
+  .debate-section {
+    height: auto;
+  }
+
+  .story-section,
+  .debate-section {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+  }
+
+  div.weather {
+    position: relative;
+    top: -90px;
+  }
+
+  .api {
+    z-index: 100;
+  }
+
+  nav.nav-navigation {
+    display: flex;
+  }
+
+  ul.navbar-nav {
+    font-size: 1.1rem;
+    line-height: 24px;
+    font-family: "Roboto Condensed", sans-serif;
+    color: black;
+  }
+
+  ul.navbar-nav1 li a,
+  ul.nav li {
+    color: black;
+    font-weight: 900;
+  }
+
+  ul.nav li,
+  ul.nav li a {
+    font: 18px/18px "Roboto Condensed", sans-serif;
+  }
+
+  ul.nav li a {
+    color: #807f84;
+    font-weight: 700;
+  }
+
+  ul.navbar-nav1 li a:hover,
+  ul.nav li a:hover {
+    color: #f72210;
+    text-decoration: none;
+  }
+
+  .debate-section {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 1rem;
+    align-self: flex-start;
+  }
+
+  .email-subscription-top {
+    width: 80%;
+    height: auto;
+    display: inline;
+  }
+
+  .form-inline {
+    width: 14%;
+    justify-content: flex-start;
```

Comment:
- [ ] There is a bootstrap class for this behavior, please use bootstrap whenever it is possible.